less/home-page.less

```diff
@@ -58,12 +80,29 @@
       }
 
       .btn {
+        .buts(center,center);
+        color: white;
+        @media @phone {
+          width: 75%;
+          order: 3;
+        }
+        &:hover {
+          background: white;
+          color: black;
+        }
       }
     }
 
     .walk-img {
```

Comment:
as for image, you can have a global style for `img` and set `width` to 100% and `height` to `auto` so it doesn't shrink like this.